compiler/luci/export/src/CircleOperationExporter.cpp

```diff
@@ -815,6 +817,48 @@ void OperationExporter::visit(luci::CircleBatchMatMul *node)
                 CreateBatchMatMulOptions(_ctx.builder, node->adj_x(), node->adj_y()).Union());
 }
 
+void OperationExporter::visit(luci::CircleBidirectionalSequenceLSTM *node)
+{
+  auto bidi_lstm_outs = loco::succs(node);
+  assert((int32_t(bidi_lstm_outs.size()) > 0) && (int32_t(bidi_lstm_outs.size()) <= 2));
```

Comment:
This is kind of explicit:)
```suggestion
  assert(bidi_lstm_outs.size() == 0 || bidi_lstm_outs.size() == 2));
```
And, IMHO, when you cast sth, it would be better to use `static_cast` for safety. BTW, why did you cast to `int32_t`? There seems no need to cast I think.

Reply:
There is no need to cast, this was more of a copy paste artifact. I will reflect the changes. Thanks.